RationaleMCP/ReadMe.md

```diff
@@ -15,49 +15,50 @@ After a release is accepted, follow the [check-list for a new release](NewReleas
 
 New MCP should be added to the following list - on the main branch to keep track of them,
 but the rest of the development on a branch/pull-request before being accepted.
+Discussion goes in the listed issue below.
```

Comment:
For some MCPs, discussions can be spread out over various PRs within the MCP, so the correctness of this statement would need to be taken with a grain of salt.  Perhaps better to not make any statement at all?

Reply:
Ok